src/main/java/org/isf/menu/manager/UserBrowsingManager.java

```diff
@@ -172,9 +172,8 @@ public void resetFailedAttempts(User user) {
 	 * Lock the {@link User} from logging into the system.
 	 *
 	 * @param user the {@link User}
-	 * @throws OHServiceException
 	 */
-	public void lockUser(User user) throws OHServiceException {
+	public void lockUser(User user) {
```

Comment:
Why are you removing the `throws`?   Please read the code where the `ioOperations` class is annotated with `@TranslateOHServiceException` which uses `OHServiceExceptionTranslator` to wrap calls to the database using `Aspects` to catch and rethrow new exceptions.   Please review all the cases where you've removed the `throws` clause.  Thanks.

Reply:
Also check any new methods you added that don't currently include the `throws` clause.

Reply:
My IDE (IntelliJ Idea) is removing them when I use the auto-clean feature, because it determines that this exception is not thrown by the method concerned. I will pay attention to this so that these Exceptions are no longer removed. 

I do not understand why we add OHServiceException on methods that do not explicitly throw this exception.
When I call a JpaRepository method like save(), all kinds of exceptions can be thrown like DataIntegrityViolationException, InvalidDataAccessResourceUsageException,..., it is not for that reason that I add all these exceptions in the signature of the method in question, but those exceptions are still raised when they're thrown by JpaRepository.

Reply:
@SilverD3 we add OHServiceException so callers knows about it and they can catch and handle them, make sense to you?

Reply:
Yeah! I see

Reply:
So are you going to restore them?

Reply:
Already!

Reply:
what do you mean?

Reply:
I was saying that I already restored them. I was asking the question just to understand.

Reply:
I'm asking because I see they are still removed on lines 188, 197 and 212 in the latest commit

Reply:
Oh sorry. I missed those ones. Thanks! It's okay now